treerec/tests/README.md

```diff
@@ -0,0 +1,67 @@
+# Test suite for tree sequence output from SLiM
+
+## Running the tests
+Just do `./run_tests.sh` from within this directory to run the tests on the results of SLiM scripts listed in the `testRecipes/` directory. Alternatively, to run just one of the recipes do `./do_test.sh testRecipes/test_XXX.slim`
+
+## Adding new tests
+
+
+To add a new test:
```

Comment:
Is the following up-to-date?

Reply:
(looks like it, thanks, I'd forgotten to update the file!)

Reply:
I think so, although I can't see how you avoid running the "don't_XXX" tests.